README.md

```diff
@@ -154,8 +154,10 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 ### • Camera
 
 * [**Open Camera**](http://opencamera.sourceforge.net/) <sup>**[[F-Droid](https://f-droid.org/app/net.sourceforge.opencamera)]**</sup>
+  * [**OpenCamera Sensors**](https://github.com/MobileRoboticsSkoltech/OpenCamera-Sensors)
```

Comment:
This looks based on *OpenCamera*, it even contains it's F-Droid link :rofl:

Reply:
> This looks based on OpenCamera

Yes, with some extra features...

> even contains it's F-Droid link

F-Droid link ??

Reply:
Its left in a Html document: https://github.com/MobileRoboticsSkoltech/OpenCamera-Sensors/blob/d7df21d0ab533162e5ca7d4fa7c1a3908065402c/_docs/info.html#L106

😄

Reply:
Thanks, Update is on the way …